### PR TITLE
Fix: Local runtime upgrade scripts

### PR DESCRIPTION
**Pull request by wischli**
# Description

* Bumps deps of `./scripts/js` to latest version, reducing peer dependency spam in the console
* Fixes upgrade script
  * Broken after bounding democracy calls and adding `PreImage` pallet in https://github.com/centrifuge/centrifuge-chain/issues/1115 as a result of https://github.com/paritytech/substrate/pull/11649
  * Also broken after adding `proofSize` in weights v2

## How to test

```
cd scripts/js/upgrade
yarn execute $path_to_wasm "test-release-v0.10.25-latest" "centrifuge-local"
```

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: scripts/js/upgrade/index.js [hunk 1]
```diff
@@ -4,16 +4,24 @@ const fs = require('fs')
 const util = require('util');
 const exec = util.promisify(require('child_process').exec);
 
+const PREIMAGE_LENGTH_BOUND = 34;
```
Comment (NunoAlexandre) [inline scripts/js/upgrade/index.js:7]: How are these values obtained or updatable based on what in the future?

**Review by NunoAlexandre — COMMENTED** [review]
[question](https://tenor.com/en-GB/view/the-office-rainn-wilson-question-gif-13906735) 👀

File: scripts/js/upgrade/index.js (hunk 1 above)
Comment (NunoAlexandre) [reply scripts/js/upgrade/index.js:7]: I.e, are these values always the same for any pre-image proposing authorising a runtime upgrade? Or do we need to keep changing them? If so, what's the best way to do that? I am 90% we don't, but just want to double check.

**Review by NunoAlexandre — APPROVED** [review]
My question is not a blocker, if you have tested these changes and the code looks fine so it's good to go for me.

File: scripts/js/upgrade/index.js [hunk 2]
```diff
@@ -4,8 +4,17 @@ const fs = require('fs')
...
-const PROPOSAL_LENGTH_BOUND = 90;
+// Needs to be >= 34
+// 32 bytes from the encoding of the H256 hashed WASM blob
+// 2 for extra stuff
+const AUTHORIZE_UPGRADE_PREIMAGE_LENGTH_BOUND = 34;
+// Needs to be >= 84
+// 39 from edemocracy.xternalProposeMajority(Lookup(H256, 34)))
+// 42 from democracy.fastTrack(H256, ...)
+// 1 from utility.batchAll
+// 2 extra
```
Comment (NunoAlexandre) [inline scripts/js/upgrade/index.js:15]: I guess the "2 extra" bytes is the first for the pallet index and the second for the extrinsic index?

**Review by NunoAlexandre — DISMISSED** [review]
Left a nugget but again not a blocker for me. Thanks for fixing this and taking the suggestions in 🏄‍♂️

File: scripts/js/upgrade/index.js (hunk 2 above)
Comment (wischli) [reply scripts/js/upgrade/index.js]: [bf413ad](https://github.com/centrifuge/centrifuge-chain/pull/1308/commits/bf413adb428b6b71903079086c076ac122924df1)

File: scripts/js/upgrade/index.js [hunk 3]
```diff
@@ -4,8 +4,17 @@ const fs = require('fs')
 const util = require('util');
 const exec = util.promisify(require('child_process').exec);
 
-const PREIMAGE_LENGTH_BOUND = 34;
-const PROPOSAL_LENGTH_BOUND = 90;
+// Needs to be >= 34
+// 32 bytes from the encoding of the H256 hashed WASM blob
+// 2 for extra stuff
+const AUTHORIZE_UPGRADE_PREIMAGE_LENGTH_BOUND = 34;
```
[the remark this answers is not in the document]
Comment (wischli) [reply scripts/js/upgrade/index.js]: Totally agree and applied in [860e281](https://github.com/centrifuge/centrifuge-chain/pull/1308/commits/860e2815219dc85458e4897fe3f2f41cf6b6bb50)

**Review by NunoAlexandre — APPROVED** [review]
❤️

**Pull request merged**